austation/code/game/objects/structures/party_palloons.dm

```diff
@@ -0,0 +1,82 @@
+/////////////////////  CONSUMABLE BALLOON  /////////////////////
+
+
+/obj/item/balloon
+	name = "deflated balloon"
+	desc = "A small, deflated balloon.  You can use it in your hand to inflate it."
```

Comment:
```suggestion
	desc = "A small, deflated balloon. You can use it in your hand to inflate it."
```

Reply:
Double space after a period is actually more correct, but sure.

Reply:
one space is correct. double spacing is a relic from the days of typewriters, and microsoft has reaffirmed that one space is the correct spacing for word processing on computers.
https://www.instructionalsolutions.com/blog/one-space-vs-two-after-period